### ISSUE TITLE
salmentions!

### ISSUE THREAD

**Issue by snarfed** [pfefferle/wordpress-semantic-linkbacks#40]
@acegiak has done awesome work implementing the salmention prototype in the upstream repo. it sounds like it might not be polished enough to merge in here, but i'd love to see that happen eventually. pretty please with sugar on top!

**Comment by acegiak** [comment]
yeah the reason I haven't done that merge is I'm a little embarrassed of the hacky nature of it but maybe it's better to merge it THEN improve it rather than the other way around.

**Comment by dshanske** [comment]
I've been looking at the code, and it is integrated into the generate comment data function. I think it would be better if the comment data function stored the parsed mf2, which is recommended in the specification, and then the salmentions code was separate.
